src/bp/ui-admin/public/index.html

```diff
@@ -1,23 +1,25 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
```

Comment:
Can you check your prettier / tslint config ? Auto-format should keep that tabulation. If I save this file after you, I get a ton of changes that adds the tabulation